config/modules/validator.py

```
@@ -0,0 +1,286 @@
+                if key.endswith(("_COOLDOWN_MINUTES", "_COOLDOWN_SECONDS")):
+                    return True  # All numeric values are valid for cooldowns
+
+                # Check minimum value constraint for non-cooldown values
+                if "min" in metadata and value < metadata["min"]:
+                    return False
+
+                # Check maximum value constraint
+                if "max" in metadata and value > metadata["max"]:
+                    return False
+
+                return True
```

Ruff

Return the negated condition directly

Inline condition

(SIM103)